src/spyglass/spikesorting/spikesorting_merge.py

```diff
@@ -50,6 +56,82 @@ class CuratedSpikeSorting(SpyglassMixin, dj.Part):  # noqa: F811
         -> CuratedSpikeSorting
         """
 
+    def get_restricted_merge_ids(
+        self,
+        key: dict,
+        sources: list = ["v0", "v1"],
+        v1_artifact: bool = True,
```

Comment:
How will this expand if there's a v2? Will that need it's own bool? I might turn this into `restrict_by_artifact`, and then for combos where that doesn't make sense, add a warning.

```python
if 'v0' in sources:
    if restrict_by_artifact:
        logger.warning('V0 does not support artifact restriction')
```